_appliance/azure/launch-an-instance.md

```diff
@@ -0,0 +1,221 @@
+---
+title: [Set up Azure for ThoughtSpot]
+keywords: Azure
+last_updated: tbd
+sidebar: mydoc_sidebar
+permalink: /:collection/:path.html
+---
+
+After you’ve determined your configuration options, you must setup your virtual
+machines (VMs) using a ThoughtSpot image for Azure.
+
+
+## About the ThoughtSpot image
+
+To provision a ThoughtSpot image in the Azure portal, you’ll need to access the
+ThoughtSpot Virtual Machine in the Azure Marketplace.
+
+The ThoughtSpot Virtual Machine comes provisioned with the custom ThoughtSpot
+image to make hosting simple. A virtual machine is a preconfigured template that
+provides the information required to launch an instance of ThoughtSpot. It
+includes the following:
+
+- A template for the root volume for the instance (for example, an operating
+system, an appliance server, and applications).
+
+The ThoughtSpot Virtual Machine has the ThoughtSpot software installed and
+configured, on an CentOS base image. Check with your ThoughtSpot contact to
+learn about the latest version of the ThoughtSpot Virtual Machine.
+
+Due to security restrictions, the ThoughtSpot Virtual Machine does not have default passwords for the
+administrator users. When you are ready to obtain the password, contact
+ThoughtSpot Support.
+
+## Set up ThoughtSpot on Azure
+
+Follow these steps to provision and set up the VMs and launch ThoughtSpot.
+
+### Prerequisites
+
+Complete these steps before launching your ThoughtSpot Virtual Machine:
+
+1. Obtain an Azure login account.
+2. Set up usage payment details with Microsoft Azure.
+3. Set up a Resource Group.
+
+### Create an instance
+1. Log in to the Azure portal.
+
+    In a browser, go to http://azure.microsoft.com, and log in to your Azure account.
+
+2. Find the [ThoughtSpot Virtual Machine on the Azure
+Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/thoughtspot-inc.thoughtspotvirtualmachine).
+
+   If the above link does not work, navigate to Azure Marketplace and search for
+   “ThoughtSpot” to find the ThoughtSpot Virtual Machine.
+
+3. Click **Get It Now**.
+4. Provide a name and password for your VM.
+5. Provide a Resource Group, by clicking on ‘existing’ and selecting one.
+6. Choose **Virtual Machine Size** and select “E64S_V3 standard”.
+7. For storage, select **Managed Disks**.
+
+### Configure Network Settings
```

Comment:
Since we are leaving users the freedom to configure networks, it would good to state
that what our requirements are and what are at users’ discretion on network configuration, it could
be sth like:

1. Thoughtspot require that nodes purchased from Azure need to be reachable to each
  other so that they can communicate and form a distributed environment.
2. Thoughtspot only require those ports to be accessible between each other nodes within
   one cluster.
3. It is your discretion to restrict public access or not for all nodes/all ports, configure what subnet, etc.  

Reply:
I've incorporated these notes, thanks. Will push to this PR soon.